documentation/3_Interoperability.md

```diff
@@ -0,0 +1,375 @@
+# Data Spaces Interoperability - How to achieve Interoperability within a Data Space and across multiple Data Spaces
+
+## Motivation for interoperability
+
+Data is one of the most valuable assets in the digital economy, but its
+potential value can only be realized if it can move and interact with
+other data to produce insights that create value. For this it must be
+possible for data to be shared and reused in a trusted way.
+Interoperability, the ability of different systems and organizations to
+exchange, understand and use data, is essential for enabling data
+sharing and creating value in data ecosystems. Data Spaces help to
+establish a common understanding of trust, provide a mechanism to
+establish sharing contracts, which include access and usage policies
+that ensure the protection and accountability of data providers and data
+consumers. As Data Spaces become more prevalent and diverse, there is
+also a growing need for intra- and cross-Data Spaces interoperability.
+
+But what is a Data Spaces exactly? As we have established in the previous
```

Comment:
Are data spaces or what is a data space. But I would generally not define data spaces again here.

Reply:
This very short intro is not about defining again what a data space is, but rather calling out the characteristics of a data space that are relevant for the discussion about interoperability. I want to make it bullet proof that people can follow the line of arguments presented here.